blueprints/hr_assistant/unsupported.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+"""This module contains the dialogue states for the 'unsupported' domain in
+the MindMeld HR assistant blueprint application
+"""
+
+from .root import app
+import random
+
+
+@app.handle(intent='unsupported')
+def unsupported(request, responder):
+    query = random.choice(["males fired in 2014", "show me all married employees",
```

Comment:
Nit: make sure that all the sentences are correctly capitalized with proper diction:
"Male employees fired in 2014."
"Show me all married employees."
"Please tell me the average salary for employees."